### ISSUE TITLE
bitrateselect + progressive: does not autoPlay default item

### ISSUE THREAD

**Issue by phloxic** [flowplayer/flash#141]
It's clear that this means a restart of the clip at bitrate switch.
But should it work?
Or is bitrateselect + progressive download not supported at all?

I believe it worked at some point because I have this demo:
http://flowplayer.blacktrash.org/test/brselect-progressive.html
but currently it does not.

See also: http://flowplayer.org/forum/#!/setup_configuration#two-bitrates-via-two-differ

**Comment by phloxic** [comment]
Actually it works, except for autoPlay, it seems.

**Comment by danrossi** [comment]
i can see it changing the files for me.

**Comment by phloxic** [comment]
Yes, it works - the only minor bug is that it behaves like `autoPlay: false`.

**Comment by danrossi** [comment]
It's doing a hard reset yes because it cannot seek.

**Comment by phloxic** [comment]
The hard reset is expected, no problem.
What it doesn't do is autoPlay the default bitrate item.
imho a minor bug, but a bug.

Will update bug title.

**Comment by phloxic** [comment]
@danrossi - just to be perfectly clear: I don't think this is urgent or important, can probably be worked around in most situations, and using bitrateselect with progressive download is a rather rare scenario anyway.

**Comment by danrossi** [comment]
it's automatically playing the next file. its not calling the "play" method as in acting like a playlist its doing a reset on the stream internally so it keeps playing. Is that not intended ?

**Comment by phloxic** [comment]
???
Normally Flowplayer autoplays - unless you set `autoPlay: false`, bitrateselect does not change this behaviour: http://flowplayer.blacktrash.org/test/bitrateselect.html
But http://flowplayer.blacktrash.org/test/brselect-progressive.html does not autoplay.

**Comment by danrossi** [comment]
freaky the first example should not be autoplaying.

**Comment by danrossi** [comment]
not happening with dev flowplayer

**Comment by phloxic** [comment]
Why should the first example not be autoplaying?

**Comment by danrossi** [comment]
I was trying to figure out what you meant it's ok. "not happening with dev flowplayer"

**Comment by danrossi** [comment]
moving to ready for testing was ok on the dev i believe.
